components/cd-hr/README.md

```diff
@@ -0,0 +1,7 @@
+# Separator Component
```

Comment:
Typically I have been using the same name for the component folder and for its name. This make it easier for everyone to know we are all talking about the same thing.
So maybe `cd-separator` as `cd-hr` implies an awareness of HTML elements?